smartapps/ethayer/keypad.src/keypad.groovy

```diff
@@ -208,7 +220,7 @@ def execRoutine() {
   def armMode = atomicState.armMode
   def userApp = parent.keypadMatchingUser(atomicState.codeEntered)
 
-  sendSHMEvent(armMode)
+//  sendSHMEvent(armMode)
```

Comment:
I believe by commenting this out, you're making it so the user cannot use the device as a typical security panel.